nix/pkgs/default.nix

```diff
@@ -31,6 +31,7 @@ stdenv.mkDerivation {
       autoconf
       autogen
       automake
+      curl
```

Comment:
I think we no longer need this `curl` dependency btw

Reply:
Ah, I forgot to remove it when switching to the vendored version of `utoipa-swagger-ui`. Thanks for catching that!